float64[3] pos
float64[3] dpos
float64[3] pos_a
float64[3] dpos_a
float64[3] ddpos_a
float64[3] pos_d
float64[3] dpos_d

float64[3] pos_track_error
float64[3] ori_track_error

float64[6] F_sensor
float64[6] F_sensor_fil

float64[3] D
float64[3] K
